python/tvm/relay/adt.py

```diff
@@ -186,18 +186,23 @@ def __init__(self, lhs, rhs):
 class Match(Expr):
     """Pattern matching expression in Relay."""
 
-    def __init__(self, data, clauses):
+    def __init__(self, data, clauses, complete=True):
         """Construct a Match.
 
         Parameters
         ----------
         data: tvm.relay.Expr
             The value being deconstructed and matched.
+
         clauses: List[tvm.relay.Clause]
             The pattern match clauses.
+
+        complete: Optional[Bool]
+            Is the pattern complete(cover all cases)?
```

Comment:
```suggestion
            Should the match be complete (cover all cases)?
            If yes, the type checker will warn about any missing cases 
```
Analogous to my other comment

Reply:
It is now an error. Can you change the warn to the error? I sort of dont have a enviroment right now... Thx.

Reply:
I updated my suggestions